examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -5614,7 +5606,7 @@
       "endpointTypeName": "Anonymous Endpoint Type",
       "endpointTypeIndex": 1,
       "profileId": "0x0103",
-      "endpointId": 1,
+      "endpointId": 2,
```

Comment:
Why ?

Reply:
Must be because the two endpoint had the same id of 1. ZAP doesn't auto-increment endpoint id when adding a new one.